### DISCUSSION TITLE
Move window only on startup execution

### DISCUSSION THREAD

**Discussion by cstmth** [i3/i3#4817] [Q&A]
Hello! :) I want to automatically start my Browser (google-chrome) on startup and want it to go on my second workspace $ws2. The issue is that I only want the startup browser window put there automatically. I want to be able to open browsers in other workspaces as well without them being moved.

I am fairly new to i3 and could not directly find something in the docs. If I missed something sorry. Otherwise I'd be happy if someone could help me out. :)

**Comment by RayZ0rr** [comment]
Look at the code snippet just above [4.19. Automatically starting applications on i3 startup](https://i3wm.org/docs/userguide.html#_automatically_starting_applications_on_i3_startup)

Basically, this command `exec --no-startup-id i3-msg 'workspace $ws2; exec chrome; workspace 1'`
change `chrome` with whatever command you use to start chrome and put this in your i3 config.

**Reply by Airblader** [reply]
Small note: for this to work the application must support startup notifications.

Otherwise this can also be achieved by using layout saving and restoring.

**Reply by cstmth** [reply]
Thanks for your help. I got the following right here:

```
exec_always i3-msg "workspace $ws8; exec evolution"
exec_always i3-msg "workspace $ws8; exec google-chrome --new-window https://calendar.google.com"
exec_always discord
exec_always i3-msg "workspace $ws10; exec spotify"
exec_always i3-msg "workspace $ws10; exec blueman-manager"
```

It's set to exec_always for debugging purposes. It works flawlessly except for line 2. It does not show up on ws8 but always somewhere else (often times ws10). Does this have something to do with the aforementioned startup notifications and if so, how can I get around it?

**Reply by Airblader** [reply]
I have no idea if Chrome supports them or not, but 

> how can I get around it?

See my reply above.
